frontends/infinite-corridor/src/test-utils/index.tsx

```diff
@@ -58,34 +58,39 @@ const renderWithProviders = (
 
 /**
  * Assert that a functional component was called at some point with the given
- * props. Optionally, specify a call index.
- * Optionally, specify that a particular function number occured with given props.
+ * props.
  * @param fc the mock or spied upon functional component
  * @param partialProps an object of props
- * @param nthCall Optional index. If specified, then asserts that particular
- *  call occurred with given props.
  */
 const expectProps = (
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   fc: jest.Mock<any, any>,
-  partialProps: unknown,
-  nthCall?: number
```

Comment:
I removed the `nthCall` argument. It doesn't make much sense when running in StrictMode (since StrictMode calls component render functions multiple times).

Additionally, we almost always used it with `nthCall = -1`. That value ("previous call") still makes sense, so now `expectLastProps` exists.